src/bundle.ts

```diff
@@ -32,14 +36,22 @@ export class BuildBundler extends Transform {
   bundler: Bundler;
   sharedFile: File;
 
+  files = new Map<string, File>();
+
   constructor(config: ProjectConfig, analyzer: BuildAnalyzer) {
     super({objectMode: true});
 
     this.config = config;
+
     this.analyzer = analyzer;
     this.sharedBundleUrl = 'shared-bundle.html';
     this.bundler = new Bundler({
-      analyzer: analyzer.analyzer,
+
+      // TODO(usergenic): Creating a new Analyzer with a blank cache is going
+      // to mean, at least, a doubling of analysis efforts for bundling phase.
+      // Ideally we would fork existing analyzer and replace its urlLoader if
+      // there were an affordance to do so.
+      analyzer: new Analyzer({urlLoader: new StreamLoader(this)}),
```

Comment:
use the newly released _fork()?

Or maybe, because it's an underscore method and so kinda experimental, it would be better to do something like

```typescript
let localAnalyzer;
if (analyzer._fork) {
  localAnalyzer = analyzer._fork();
  // do something to modify the url loader
} else {
  localAnalyzer = new Analyzer({/*...*/});
}